nemo/collections/multimodal/models/multimodal_llm/neva/neva_model.py

```diff
@@ -613,16 +613,16 @@ def forward(self, tokens, text_position_ids, attention_mask, labels, media=None)
         output_tensor = self.model(**forward_args)
         return output_tensor
 
-    def fwd_bwd_step(self, dataloader_iter, batch_idx, forward_only, first_val_step=None):
-        return MegatronGPTModel.fwd_bwd_step(self, dataloader_iter, batch_idx, forward_only, first_val_step)
+    def fwd_bwd_step(self, dataloader_iter, forward_only, first_val_step=None):
+        return MegatronGPTModel.fwd_bwd_step(self, dataloader_iter, forward_only, first_val_step)
 
-    def training_step(self, dataloader_iter, batch_idx):
+    def training_step(self, dataloader_iter):
         """
```

Comment:
PTL 2.1 introduces change in the training/validation/test_step signature while using `dataloader_iter` API. Removes `batch_idx` from the signature and `dataloader_iter` returns a triplet` (batch, batch_idx, dataloader_idx)`: [More details here](https://lightning.ai/docs/pytorch/stable/upgrade/from_2_0.html#:~:text=training_step(self%2C%20dataloader_iter%2C%20batch_idx)%20in%20LightningModule-,remove%20batch_idx%20from%20the%20signature%20and%20expect%20dataloader_iter%20to%20return%20a%20triplet%20(batch%2C%20batch_idx%2C%20dataloader_idx),-PR18390)